docs/pages/components/form.md

```diff
@@ -32,12 +32,11 @@ Do not use the input field if:
 - The user needs to carry out a search. In this case, use the search field.
 
 {% capture inputs %} 
-    <div class="fd-form-item">
-        <label class="fd-form-label" for="input-1a">Default Input:</label>
-        <input class="fd-input" type="text" id="input-1a" placeholder="Field placeholder text">
+    <div class="fd-form-item fd-has-margin-none fd-popover">
+        <label class="fd-form-label" for="input-1">Default input:</label>
+        <input class="fd-input fd-popover__control" type="text" id="input-1" placeholder="Field placeholder text">
     </div>
```

Comment:
Why do you put `fd-popover` and `fd-popover__control` when you don't have a message?

Reply:
It was supposed to be removed.There used to be a form-message on that item. I'll remove em.